jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfigurationTest.kt

```diff
@@ -103,6 +103,7 @@ class LocalLambdaRunConfigurationTest {
     @Test
     fun samIsNotSet() {
         val fakeSamPath = "NotValid"
+        preWarmLambdaHandlerValidation(projectRule.project)
```

Comment:
Why is this now needed? add a comment?

Reply:
Sam is validated after the handler is validated (so we can get the runtime)